build-system/babel-plugins/babel-plugin-jsx-style-object/test/fixtures/transform/dimensional/output.mjs

```diff
@@ -0,0 +1,136 @@
+import { jsxStylePropertyString as _jsxStylePropertyString44 } from "#core/dom/jsx-style-property-string";
```

Comment:
Continuing https://github.com/ampproject/amphtml/pull/36890#discussion_r747001970 (which is outdated and so I can't comment directly anymore):

This is fine, since it's just build output. The minifier will coalesce these into a single import value.